code/object/collideshipweapon.cpp

```diff
@@ -66,7 +66,7 @@ static void update_danger_weapon(object *pship_obj, object *weapon_obj)
  * Deal with weapon-ship hit stuff.  
  * Separated from check_collision routine below because of multiplayer reasons.
  */
-static void ship_weapon_do_hit_stuff(object *pship_obj, object *weapon_obj, vec3d *world_hitpos, vec3d *hitpos, int quadrant_num, int submodel_num, vec3d /*not a pointer intentionaly*/ hit_dir)
+static void ship_weapon_do_hit_stuff(object *pship_obj, object *weapon_obj, vec3d *world_hitpos, vec3d *hitpos, int quadrant_num, int submodel_num, vec3d *hit_dir)
```

Comment:
If this can safely be pointered, then it should definitely also be consted to indicate that the vector isn't modified.

Reply:
Agreed, but I figured I'd defer that to the upcoming PR dedicated to `const`.  I can still add `const` here if you'd rather I do this particular one now instead of later.

Reply:
Mhm, it's fine if said const PR is gonna come soon.
Otherwise, having a non-const pointer is kinda ugly

Reply:
Ok, done